### ISSUE TITLE
3.x-dev - reduced CMS accessibility due to low contrast

### ISSUE THREAD

**Issue by kinglozzer** [silverstripe/silverstripe-framework#4860]
3.2 labels/descriptions:
![screen shot 2015-12-21 at 16 54 35](https://cloud.githubusercontent.com/assets/1655548/11936125/c166377c-a803-11e5-9d68-88c85a4872a1.png)

3.x (3.3) labels/descriptions:
![screen shot 2015-12-21 at 16 54 09](https://cloud.githubusercontent.com/assets/1655548/11936129/c6731d0c-a803-11e5-87e5-d9b122510c59.png)

When in doubt, I use a colour contrast checker:

- [3.2 labels](http://snook.ca/technical/colour_contrast/colour.html#fg=444444,bg=E6EAED) - Fully WCAG 2 AA and AAA compliant
- [3.2 descriptions](http://snook.ca/technical/colour_contrast/colour.html#fg=777777,bg=E6EAED) - WCAG 2 AA compliant at 18pt+ only (i.e. fail, as font is too small)
- [3.x labels](http://snook.ca/technical/colour_contrast/colour.html#fg=66727D,bg=ECEFF1) - WCAG 2 AA compliant at 18pt+ only (i.e. fail, as font is too small)
- [3.x descriptions](http://snook.ca/technical/colour_contrast/colour.html#fg=9BA5AE,bg=ECEFF1) - not compliant

Descriptions are arguably non-essential, but I think we should at least darken the field labels and input colours a bit to ensure the critical parts of the CMS are accessible. The same issue applies to tabs and “Add new page” button text too.

**Comment by kinglozzer** [comment]
Pinging @clarkepaul @scott1702 @flashbackzoo @jonom for input :)

**Comment by jonom** [comment]
That's a neat tool. I too find text in general a *little* bit too washed out in 3.x. since it's on a relatively dark background. The only element type that stands out to me is the description, they're fairly hard to read. The rest I wouldn't be too bothered if they stayed how they are but I would prefer them to be a touch darker.

In terms of WCAG 2AA compliance I don't know too much about it but I would assume for our purposes it's okay if we're meeting targets "at 18pt+ only" as users can increase the font-size by zooming in the browser?

**Comment by flashbackzoo** [comment]
We should be aiming to improve accessibility every release (or at least not make it worse). I've added a ticket to our internal backlog :smile:

**Comment by sminnee** [comment]
Marking as a 'patch' change since it's a UX bugfix. Expect to happen in `3.3` branch after the (imminent) release of 3.3-beta1

**Comment by kinglozzer** [comment]
Thanks all :+1:

**Comment by tractorcow** [comment]
Please push to the 3.3 branch in order for it to be ready for the upcoming rc1

**Comment by tractorcow** [comment]
Fixed with #4885
